Svenbjorn/whatson
# Minor changes to UI on place fonts and positioning

# Minor changes to UI [a810c6a]

## Details

- Fonts and positioning fixed
- I noticed that the details side-view breaks apart and also changes order when resizing the browser window
    - This will have to be fixed in another commit

## Testing

- Open a place and see if the look matches the layout from [Sendistovan](https://sendistovan.invisionapp.com/console/share/HYFENMAGVB8/985474050/play)
- See if anything breaks when switching to mobile view
- Compare to notes from James in monday?
